### PR TITLE
Unfocus Entry on Enter pressed

### PR DESCRIPTION
**Pull request by JKennedy24**
### Description of Change ###

Unfocus UWP Entry on Enter Pressed

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Rebased on top of master at time of PR
- [ ] Changes adhere to coding standard
- [ ] Consolidate commits as makes sense

### CODE REVIEW

**Comment by rmarinho** [comment]
@ez can you see how this works with your fix #747

File: Xamarin.Forms.Platform.WinRT/EntryRenderer.cs [hunk 1]
```diff
@@ -111,6 +111,7 @@ void TextBoxOnKeyUp(object sender, KeyRoutedEventArgs args)
 				return;
 
 			((IEntryController)Element).SendCompleted();
+			Control?.Unfocus();
```
Comment (hartez) [inline Xamarin.Forms.Platform.WinRT/EntryRenderer.cs]: Why do you want the TextBox to lose focus when you hit Enter?

Comment (JKennedy24) [inline Xamarin.Forms.Platform.WinRT/EntryRenderer.cs]: These Entries are used mostly for Data Entry Forms / User Name / Password boxes.
The Editor is used for Multi Line - Where Enter is handled differently

To me Enter means "Complete - I'm finished with this text box" So the logical thing to mean was to unfocus the box.

**Comment by StephaneDelcroix** [comment]
Part of this issue is fixed in #747. the other part is not valid (according to @hartez )

**Pull request closed without merging**